app/code/core/Mage/Eav/etc/config.xml

```diff
@@ -132,6 +132,18 @@
             </modules>
         </translate>
     </frontend>
+    <crontab>
+        <jobs>
+            <clean_orphaned_records>
+                <schedule>
+                    <cron_expr>0 2 * * *</cron_expr>
```

Comment:
I've used the same schedule as the catalog_product_index_price_reindex_all, is that ok?

Reply:
I think it should be `0 0 * * *`, the other two events that happen at midnight are `sales_clean_quotes` and `core_email_queue_clean_up`, so it seems the time to clean things up.

I also think midnight is good because at 01:00 we run `catalogrule_apply_all` and at 02:00 we run `catalog_product_index_price_reindex_all`. Since both of those two events certainly use attributes, best not do the cleaning at the same time.